content/geometry/object-vehicle/vehicle-structure-model-structure.adoc

```diff
@@ -5,68 +5,82 @@
 [plantuml]
 ----
 legend
-Vehicle structure
-|_ Grp_Vehicle_Part_<keyword>_<vehicle_part_idx> (T)
-  |_ Grp_Exterior
-    |_ Grp_Exterior_Static
-    |_ Grp_Exterior_Dynamic
-      |_ Grp_Wheel_<axle_idx>_<wheel_idx> (T)
-        |_ Grp_Wheel_<axle_idx>_<wheel_idx>_Steering
-        |_ Grp_Wheel_<axle_idx>_<wheel_idx>_Steering_Rotating (T)
-      |_ Grp_Door_Front_<door_front_idx> (T)
-      |_ Grp_Door_Left_<door_left_idx> (T)
-      |_ Grp_Door_Right_<door_right_idx> (T)
-      |_ Grp_Door_Rear_<door_rear_idx> (T)
-      |_ Grp_Door_Top_<door_top_idx> (T)
-      |_ Grp_Door_Bottom_<door_bottom_idx> (T)
-      |_ Grp_Light_Day_Left_<day_left_idx> (T)
-      |_ Grp_Light_Day_Right_<day_right_idx> (T)
-      |_ Grp_Light_Low_Beam_Left_<low_beam_left_idx> (T)
-      |_ Grp_Light_Low_Beam_Right_<low_beam_right_idx> (T)
-      |_ Grp_Light_High_Beam_Left_<high_beam_left_idx> (T)
-      |_ Grp_Light_High_Beam_Right_<high_beam_right_idx> (T)
-      |_ Grp_Light_Position_Left_<position_left_idx> (T)
-      |_ Grp_Light_Position_Right_<position_right_idx> (T)
-      |_ Grp_Light_Park_Left_<position_left_idx> (T)
-      |_ Grp_Light_Park_Right_<position_right_idx> (T)
-      |_ Grp_Light_Tail_Left_<tail_left_idx> (T)
-      |_ Grp_Light_Tail_Right_<tail_right_idx> (T)
-      |_ Grp_Light_Brake_Left_<brake_left_idx> (T)
-      |_ Grp_Light_Brake_Center_<brake_center_idx> (T)
-      |_ Grp_Light_Brake_Right_<brake_right_idx> (T)
-      |_ Grp_Light_Reverse_Left_<reverse_left_idx> (T)
-      |_ Grp_Light_Reverse_Right_<reverse_right_idx> (T)
-      |_ Grp_Light_Fog_Left_<fog_left_idx> (T)
-      |_ Grp_Light_Fog_Right_<fog_right_idx> (T)
-      |_ Grp_Light_Corner_Left_<corner_left_idx> (T)
-      |_ Grp_Light_Corner_Right_<corner_right_idx> (T)
-      |_ Grp_Light_Indicator_Left_<indicator_left_idx> (T)
-      |_ Grp_Light_Indicator_Right_<indicator_right_idx> (T)
-      |_ Grp_Number_Plate_<number_plate_idx> (T)
-      |_ Grp_Light_Number_Plate_<number_plate_light_idx> (T)
-      |_ Grp_Light_Warning_<warning_idx> (T)
-      |_ Grp_Convertible_Top
-      |_ Grp_Sensors (T)
-      |_ Grp_Side_Mirror_Mounting_Left_<side_mirror_mounting_left_idx> (T)
-      |_ Grp_Side_Mirror_View_Left_<side_mirror_view_left_idx> (T)
-      |_ Grp_Side_Mirror_Mounting_Right_<side_mirror_mounting_right_idx> (T)
-      |_ Grp_Side_Mirror_View_Right_<side_mirror_view_right_idx> (T)
-      |_ Grp_Blindspot_Mirror_Mounting_<blindspot_mirror_mounting_idx> (T)
-      |_ Grp_Blindspot_Mirror_View_<blindspot_mirror_view_idx> (T)
-  |_ Grp_Interior
-    |_ Grp_Interior_Static
-    |_ Grp_Interior_Dynamic
-      |_ Grp_Steering_Wheel (T)
-      |_ Grp_Eyepoint_<eyepoint_idx> (T)
-      |_ Grp_Rearview_Mirror_Mounting_<rearview_mirror_mounting_idx> (T)
-      |_ Grp_Rearview_Mirror_View_<rearview_mirror_view_idx> (T)
-      |_ Grp_Seat_<seat_idx> (T)
+Root
```

Comment:
AFAIK this is still up for discussion.
In legacy openMATERIAL, the structure was 

Root
|-Exterior
|-Interior
|-Floor

So the root node had its purpose as a representation of the vehicle.

With the introducted of vehicle parts, if we deem the latter as self-contained, we could have them at the topmost hierarchy level.
Personally, I'd favor to have a "root" node as group node for the vehicle parts and as representation of the vehicle since I can't imagine using the vehicle parts stand-alone.

Reply:
We decided that every object should have a root node. This explanation should go into a general chapter, as it is true for all object classes.